Source/Imaginary.swift

```diff
@@ -31,14 +31,16 @@ public struct Imaginary {
 }
 
 public var imageCache: Cache<UIImage> {
+
   struct Static {
     static let config = Config(
       frontKind: .Memory,
       backKind: .Disk,
       expiry: .Date(NSDate().dateByAddingTimeInterval(60 * 60 * 24 * 3)),
-      maxSize: 0)
+      maxSize: 0,
+      maxObjects: 10)
```

Comment:
This should probably be configurable in the future, but right now 10 felt like a good amount.